specification/dimensions/billingcurrency.md

```diff
@@ -0,0 +1,76 @@
+### Billing Currency
+
+Billing Currency is an identifier that represents the currency that a charge for resources and/or services was billed in. Billing Currency is commonly used in scenarios where costs need to be grouped or aggregated.
+
+The BillingCurrency column MUST be present in the billing data. BillingCurrency MUST match the currency used in the invoice generated by the invoice issuer. This column must be of type String and MUST NOT contain null values. BillingCurrency MUST be one of the values supported in [ISO 4217:2015](https://www.iso.org/standard/64758.html).
+
+[Attribute: Currency Code Format](link) contains further details about formatting requirements for currency values.
```

Comment:
based on the title of the attribute, you can put the link to the anchor here. The build process just concatenates the words (see other places we link).

Reply:
Okay I will push a commit for the attribute first then I'll be able to better link it here

Reply:
no need. You just need to know the value. Markdown conversion keeps the same format for it.